e2e/tests/workflows.spec.ts

```diff
@@ -17,8 +17,9 @@ test.describe("Workflows list", () => {
       .first()
       .click({ position: { x: 0, y: 0 } });
 
-    await expect(page.locator("text=Workflow Type").first()).toBeVisible();
-    await expect(page.locator("text=Task Queue").first()).toBeVisible();
+    await expect(page.locator("text=Summary").first()).toBeVisible();
```

Comment:
Actually, looks like it's [not finding it](https://github.com/temporalio/ui-server/actions/runs/3997426190/jobs/6858744760#step:11:539) 😞